vignettes/implementation.Rmd

```diff
@@ -168,28 +170,68 @@ grts$object %>%
   coord_fixed()
```

Comment:
I did a few (manual) iterations here as in this vignette figure only few points with `ffreq` = 3 are retained in the sample, compared to the input. In my iterations, the number of points with `ffreq` = 3 varied quite a lot, but mostly it was a low number (< 5), and sometimes being zero! If this is normal behaviour, some explanation is needed here.